src/util/tempReviews.ts

```diff
@@ -1,8 +1,6 @@
-import { Review, User, Venue } from "API";
-
 export class TempReviews {
-    public static getTempReviews(): Review[] {
```

Comment:
I know I made this file but we shouldn't put much work into maintaining it. It only exists as a placeholder content until we start talking to the API.